lib/api.js

```diff
@@ -32,7 +32,7 @@ function loginByEmail() {
   return api
     .post("auth/v2/loginByEmail", {
       json: {
-        device_type: "UNKNOWN",
+        device_type: "ANDROID",
```

Comment:
Maybe : device_type: credentials.device_type, and put "device_type": "ANDROID", in config.json